### ISSUE TITLE
Item highlighting

### ISSUE THREAD

**Issue by Morik** [d07RiV/diabloweb#65]
If you follow the link to the devilution repo you find a nice modded screenshots where they added monster lifebar and item highlighting.
I would love to have that also in the web version as it is hard to see the items especially on my smartphone...

**Comment by sskras** [comment]
No, they didn't add this feature yet -- it was only a PoC, I believe, made by @galaxyhaxz as an example of what could be done.

And that should go either into [diasurgical/devilutionX](https://github.com/diasurgical/devilutionX/) or into [diasurgical/devilution-plus-plus](https://github.com/diasurgical/devilution-plus-plus). 

The latter is declared to be "This is Diablo / devilution With added features and fixes for balance."  But the development has stalled alread for a year.

**Comment by Morik** [comment]
OK, thanks for your answer, so i guess manually implementation isn't possible ?

**Comment by d07RiV** [comment]
I could try, finding items on a phone has been a big pain. But I haven't really dug into the actual gameplay code so I'm not sure if it's possible to make a nice looking effect for that.

**Comment by galaxyhaxz** [comment]
Adding such a feature is quite easy. The only challenging part is stacking item names so they dont appear on top of each other. Basically you just add code in scrollrt.cpp when the items are being drawn.

Monster lifebar is easier. Diablo 2 stores are a bit tricker and require more code and new graphics.

**Comment by Morik** [comment]
Maybe you could submit a pull request to the webassembly repo ?

**Comment by galaxyhaxz** [comment]
Don't feel like it, got too much stuff on my hands right now. A great way to get into the project would be to study and understand the code so you could write such a thing yourself!
